src/libpriv/rpmostree-postprocess.c

```diff
@@ -1486,20 +1502,11 @@ rpmostree_treefile_postprocessing (GFile         *yumroot,
                        "'remove' elements must be relative");
           goto out;
         }
+      g_assert (val[0] != '/');
+      g_assert (strstr (val, "..") == NULL);
 
-      child = g_file_resolve_relative_path (yumroot, val);
-        
-      if (g_file_query_exists (child, NULL))
-        {
-          g_print ("Removing '%s'\n", val);
```

Comment:
Any reason for removing this? I find it a useful msg to help sanity checking and investigating composes gone awry. Maybe tweak it to e.g. `"Ensuring deletion of '%s'\n"` so we don't lie if it doesn't exist.